get-started/faq/index.md

```diff
@@ -124,3 +124,17 @@ Fungibility is a simple property of money such that there are no differences bet
 </div>
 
 </div>
+
+<div class="tab">
+    <input id="tab-eleven" type="checkbox" name="tabs" class="accordian">
+    <label for="tab-eleven" class="accordian">If Monero is so private how do we know they're not being created out of thin air?</label>
+    
+<div class="tab-content" markdown="1">
+In Monero, every transaction generates a key image. Key images that are used more than once are rejected by the miners as double-spends and cannot be added to a valid block. When a new transaction is received, miners verify that the key image does not already exist for a previous transaction to ensure it's not a double-spend.
```

Comment:
> every transaction generates a key image

This is a bit inaccurate technically. Key image can be generated only by the recipient when he scans the blockchain. Perhaps reword it like, `every transaction output is uniquely associated with a key image that can only be generated by the holder of that output.`